.vscode/extensions.json

```diff
@@ -4,6 +4,7 @@
     "esbenp.prettier-vscode",
     "donjayamanne.git-extension-pack",
     "msjsdiag.debugger-for-chrome",
+    "msjsdiag.debugger-for-edge",
```

Comment:
This proposes to a developer when they open Visual Studio if the would like to install the Edge Debugger.